db/schema.rb

```diff
@@ -197,6 +197,10 @@
     t.text "resignation_reason"
     t.text "rematch_reason"
     t.boolean "gdpr_consent", default: false
+    t.boolean "like_mentoring_beginner", default: false
```

Comment:
what does this field do ? Can we add comments on column that we add ?

Reply:
Added comment